nimajalali/triton-kubernetes
# Add vendor dependencies

## How to Test
1. Make sure you have dep installed. `brew install dep`
2. Checkout this branch
3. Run `dep ensure` to create the `vendor/` directory locally.
4. Run `go build` and try out a few commands.

## Notes

* **Vendor Directory Size**
  * When you run `dep ensure` you also need to run `dep prune` to get rid of any subpackages that aren't being used.
  * For `triton-kubernetes` running `dep ensure` creates a ~270M vendor directory. Running `dep prune` reduced it to 23M
* **Set revision hash for github.com/manifoldco/promptui to 4baa118**
  * Our source code depends on the Prompt stdin field to be of type io.ReadCloser. The current release of promptui (v0.2.1) does not have this, so we have to use the commit hash. This should be changed to a version once promptui makes a new release.
